### ISSUE TITLE
adjustText breaking changes

### ISSUE THREAD

**Issue by SarahNadeau** [ygidtu/trackplot#96]
I think trackplot's call to adjust_text has arguments that are unfortunately not supported in the newest version of the adjustText package.

Here is the call: https://github.com/ygidtu/trackplot/blob/main/trackplot/plot_func.py#L810
Here are the adjustText docs where you can see that e.g. the 'autoalign' argument doesn't exist anymore: https://adjusttext.readthedocs.io/en/latest/. 
Here is a related issue where adjustText references the breaking changes: https://github.com/Phlya/adjustText/issues/167

As a workaround, I just pinned adjustText to be the older version in my trackplot conda environment. This is the environment yaml file that is working for me:
```
name: trackplot
channels:
  - conda-forge
  - bioconda
dependencies:
  - trackplot
  - adjusttext<1
  - pygtftk
```

**Comment by ygidtu** [comment]
Hi Sarah,
   I released a new version v0.3.8, fix the issues with the `adjust_text`. This version was tested with `--show-junction-num` and `--motif`, problem should be solved. please try the latest version and feel free to contact us.
Best
Yiming

**Comment by ygidtu** [comment]
Hi Sarah,
    A while since I haven't heard your feedback, I am temporarily closing this issue. If you have any questions, please feel free to contact me at any time.
Best,
Yiming
